### PR TITLE
Feature/125

### PR DESCRIPTION
**Pull request by dchertousov**
Added importPNR for bookings with cancelled segments

### CODE REVIEW

File: src/Services/Air/Air.js [hunk 1]
```diff
@@ -39,7 +39,74 @@ module.exports = (settings) => {
     },
 
     importPNR(options) {
-      return service.importPNR(options);
+      return service.importPNR(options)
+        .catch((err) => {
+          // Checking for error type
+          if (
+              !(err instanceof AirRuntimeError) ||
+              (err.data.faultstring !== 'No provider reservation to import.')
```
Comment (shmuga) [inline src/Services/Air/Air.js]: Should be parsed as separate error inside AirError.

File: src/Services/Air/Air.js [hunk 2]
```diff
@@ -39,7 +39,74 @@ module.exports = (settings) => {
...
+          ) {
+            return Promise.reject(err);
+          }
+          // Creating passive segment to import PNR
+          const terminal = createTerminalService(settings);
+          const segment = {
```
Comment (shmuga) [inline src/Services/Air/Air.js]: Let's move this segment creation to utils as function `getPassiveSegmentCommand` or something like that.

Comment (shmuga) [inline src/Services/Air/Air.js]: I mean with segmentCommad and segmentResult.

Comment (shmuga) [inline src/Services/Air/Air.js]: Or maybe event better to create folder with make command helpers.

File: src/Services/Air/Air.js [hunk 3]
```diff
@@ -39,7 +39,74 @@ module.exports = (settings) => {
...
+            comment: 'NO1',
+            class: 'Y',
+          };
+          const segmentCommand = (
+            `0${segment.airline}OPEN${segment.class}${segment.date}${segment.from}${segment.to}${segment.comment}`
+          ).toUpperCase();
+          const segmentResult = (
+            `1. ${segment.airline} OPEN ${segment.class}  ${segment.date} ${segment.from}${segment.to} ${segment.comment}`
+          ).toUpperCase();
+          const pnrRegExp = new RegExp(`^(?:\\*\\* THIS BF IS CURRENTLY IN USE \\*\\*\\s*)?${options.pnr}`);
```
Comment (shmuga) [inline src/Services/Air/Air.js]: Please move to separate parser in utils.

Comment (shmuga) [inline src/Services/Air/Air.js]: It's better to test it there.

File: src/Services/Air/Air.js (hunk 1 above)
Comment (dchertousov) [inline src/Services/Air/Air.js]: OK

**Comment by dchertousov** [comment]
@shmuga Please review updates

**Comment by shmuga** [comment]
Great job 👍

**Comment by shmuga** [comment]
Closes #125

**Pull request merged**